code/datums/martial/reverbpalm.dm

```diff
@@ -137,14 +137,22 @@
 	COOLDOWN_START(src, next_suplex, COOLDOWN_SUPLEX)
 	footsies(target)
 	var/turf/Q = get_step(get_turf(user), turn(user.dir,180))
-	user.visible_message(span_warning("[user] outstretches [user.p_their()] arm and goes for a grab!"))
 	wakeup(target)
 	for(var/mob/living/L in Q.contents)
 		damagefilter(user, L, simpledam, persondam, borgdam)
 	if(!(crash(target, Q)))
 		target.forceMove(Z)
 	damagefilter(user, target, simpledam, persondam, borgdam)
 	if(isanimal(target))
+		if(istype(target, /mob/living/simple_animal/hostile/megafauna/bubblegum))
+			var/mob/living/simple_animal/hostile/megafauna/bubblegum/B = target
+			if(B.charging)
+				B.adjustBruteLoss(50)
+				B.charging = FALSE
```

Comment:
so, while you think this sounds cool, all it does is let bubblegum attack while doing that specific charge, it doesn't actually stop their triple charge attack, nor does it stop the charge itself

Reply:
at least, i'm pretty sure, as there's no check for it to stop
unless there's weird synergy with the forcemove of suplexes

Reply:
initially kept the charging false thing incase something broke and bubblegum was giving away free damage bonuses but i shouldn't expect that to break when it's gone now. i can't stop the charge but suplexing now places bg two tiles behind the user so the charge path is typically disjointed with the user out of harm's way.